lib/cylc/review.py

```diff
@@ -704,6 +706,7 @@ def view(self, user, suite, path, path_in_tar=None, mode=None,
         # Log files with +TZ in name end up with space instead of plus sign, so
         # put plus sign back in (https://github.com/cylc/cylc-flow/issues/4260)
         path = re.sub(r"(log\.\S+\d{2})\s(\d{2,4})$", r"\1+\2", path)
+        path = re.sub(r"(log\/config\/\d*T?\d*)\s(\d*-rose-suite.conf)", r"\1+\2", path)
```

Comment:
Adds + back into url - cherrypy apparently drops them - but I can't shed any light on the reasons.

Reply:
I think this explains it: https://stackoverflow.com/questions/14307387/navigate-to-file-with-a-plus-in-the-url